project-metrics/metrics_service/apis/travis.py

```diff
@@ -39,11 +40,7 @@ def __init__(self, travis_token: Text = None, *args, **kwargs):
     self.setClient(agithub_base.Client(*args, **kwargs))
     self.setConnectionProperties(props)
 
-    if travis_token:
-      self._token = travis_token
-    else:
-      self._token = self._get_travis_token(env.get('GITHUB_API_ACCESS_TOKEN'))
-
+    self._token = self._get_travis_token(env.get('GITHUB_API_ACCESS_TOKEN'))
```

Comment:
Document this change in the PR description

Reply:
Done.